src/app/pages/sharing/components/shares-dashboard/shares-dashboard.component.css

```diff
@@ -0,0 +1,122 @@
+.container {
+    height: 90%;
```

Comment:
It looks like indentation uses 4 spaces instead of 2. And seems we discussed using Scss syntax.